README.md

```diff
@@ -1,73 +1,24 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# README
 
-# Project Name
+This README would normally document whatever steps are necessary to get the
+application up and running.
 
-> Description the project.
+Things you may want to cover:
 
+* Ruby version
 
-## Built With
+* System dependencies
 
-- Major languages
-- Frameworks
-- Technologies used
+* Configuration
 
-## Live Demo (if available)
+* Database creation
 
-[Live Demo Link](https://livedemo.com)
+* Database initialization
 
+* How to run the test suite
 
-## Getting Started
+* Services (job queues, cache servers, search engines, etc.)
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+* Deployment instructions
```

Comment:
Kindly, make sure to use the provided template. I see you have professional `README` file in your other branches.

Reply:
thank you. this has been implimented